server/webdriver/cdc_tests/autopush_test.py

```diff
@@ -66,3 +65,12 @@ def test_statvar_explorer(self):
                                '//*[contains(text(), "Average Annual Wage")]')
     self.assertIsNotNone(custom_sv_elem,
                          'Custom SV (Average Annual Wage) element not found')
+
+
+class CdcAutopushNLTest(ExploreTest):
+
+  def get_server_url(self):
+    return CDC_AUTOPUSH_URL
+
+  def test_cdc_nl(self):
+    self.run_detect_and_fulfill('cdc_nl', ['gender wage gap in europe'])
```

Comment:
This is very nice!

As a followup in a future PR, if we can share the test-case queries, that would be wonderful.